src/core/parcel-validation.ts

```diff
@@ -0,0 +1,127 @@
+import { generateLazyValidator, JSONSchema, ValidateFunction } from '../validation'
+import { isOutOfBoundsParcel } from './parcel-exceptions'
+
+/**
+ * Represents a parcel with X and Y coordinates
+ * @public
+ */
+export type Parcel = { x: number; y: number }
+
+/**
+ * Standard limits for parcels in Decentraland
+ * @public
+ */
+export const PARCEL_LIMITS = {
+  minX: -150,
+  maxX: 150,
+  minY: -150,
+  maxY: 150
+}
+
+/**
+ * Namespace with utilities to validate parcels
+ * @public
+ */
+export namespace Parcel {
+  export const schema: JSONSchema<Parcel> = {
+    type: 'object',
+    required: ['x', 'y'],
+    properties: {
+      x: {
+        type: 'number'
+      },
+      y: {
+        type: 'number'
+      }
+    }
```

Comment:
If we want to fully use the AJV schemas, we can include a validation function that uses the functions you've created.